### PR TITLE
Updated transaction authorisation flaky test

### LINKED ISSUE #1776: e2e - flaky - should not reset attempts on cancel - non transaction UI
#### What would you like to be added:
https://dashboard.cypress.io/projects/oqk3fk/analytics/top-failures/238f4d0d-b7cf-2c56-a14a-57f45da7ac71-32adf3b7-b4c1-351d-8dba-3acd32d8378d?branches=%5B%5D&browsers=%5B%5D&chartRangeMostCommonErrors=%5B%5D&chartRangeSlowestTests=%5B%5D&chartRangeTopFailures=%5B%5D&committers=%5B%5D&cypressVersions=%5B%5D&flaky=%5B%5D&operatingSystems=%5B%5D&runGroups=%5B%5D&specFiles=%5B%5D&status=%5B%7B%22label%22%3A%22Passed%22%2C%22value%22%3A%22PASSED%22%7D%2C%7B%22label%22%3A%22Failed%22%2C%22value%22%3A%22FAILED%22%7D%5D&tags=%5B%5D&timeInterval=WEEK&timeRange=%7B%22startDate%22%3A%222021-11-21%22%2C%22endDate%22%3A%222021-12-21%22%7D&viewBy=TEST_CASE

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind chore

#### Which issue(s) does this PR fixes?:

Fixes #1776

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts [hunk 1]
```diff
@@ -33,24 +33,30 @@ context('Wallet - Transaction Authorization with Error', () => {
...
+  const MAX_PASSCODE_ATTEMPT = 3
   it('should not reset attempts on cancel - non transaction UI', function () {
     cy.createEmptyWallet(true)
     cy.getByTestID('header_settings').click()
-    cy.getByTestID('view_recovery_words').click().wait(3000)
-    Array.from(Array(MAX_PASSCODE_ATTEMPT - 1), (v, i) => i + 1).forEach(() => {
+    cy.getByTestID('view_recovery_words').click()
+    cy.wait(3000)
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for (const _i of Array(MAX_PASSCODE_ATTEMPT - 1)) {
```
Comment (thedoublejay) [inline mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:57]: What is the issue for this?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ context('Wallet - Transaction Authorization with Error', () => {
 
   it('should not reset attempts on cancel', function () {
     cy.getByTestID('button_confirm_convert').click().wait(2000)
-    Array.from(Array(MAX_PASSCODE_ATTEMPT - 1), (v, i) => i + 1).forEach(() => {
+    cy.wrap(Array(MAX_PASSCODE_ATTEMPT - 1)).each(() => {
```
Comment (kyleleow) [inline mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:36]: Curious, what was the problem if we keep `Array.from...`?

Comment (fullstackninja864) [reply mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:36]: Foreach is async function so I have replaced it by cypress loop

Comment (kyleleow) [reply mobile-app/cypress/integration/functional/wallet/authorization/transactionAuthorization.spec.ts:36]: nice catch, didnt know `forEach()` in cypress is running in parallel and not sequential, thanks!

**Pull request merged**